Services/Models/DeviceModel.cs

```diff
@@ -27,22 +27,28 @@ public Script()
 
     public class DeviceModel
     {
+        public string ETag { get; set; }
         public string Id { get; set; }
         public string Version { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
+        public string Type { get; set; }
```

Comment:
TODO: check whether we need this field at all, i.e. if the web API clients need to know whether a model is stock or not (maybe to know what can be deleted?)

Reply:
We need this field to show the type of a device model in UI.

Reply:
👍